packages/labs/gen-wrapper-react/src/index.ts

```diff
@@ -0,0 +1,200 @@
+        'tsconfig.json': tsconfigTemplate(),
+        ...wrapperFiles(analysis.packageJson, litModules),
+      },
+    };
+  } else {
+    throw new Error('No Lit components were found in this package.');
+  }
+};
+
+const wrapperFiles = (packageJson: PackageJson, litModules: LitModule[]) => {
+  const wrapperFiles: FileTree = {};
+  for (const {
+    module: {sourcePath, jsPath},
+    elements,
+  } of litModules) {
+    wrapperFiles[sourcePath] = wrapperModuleTemplate(
+      packageJson,
+      jsPath,
+      elements
+    );
+  }
+  return wrapperFiles;
+};
+
+const packageJsonTemplate = (pkgJson: PackageJson, litModules: LitModule[]) => {
+  // Refinement of package.json generation ala the TODOs below tracked in
+  // https://github.com/lit/lit/issues/2855
+
+  // TODO(kschaaf): spread in/adapt other relevant fields from source
+  // package.json (description, license, keywords, etc.)
+  return JSON.stringify(
+    {
+      name: packageNameToReactPackageName(pkgJson.name!),
+      type: 'module',
+      scripts: {
+        build: 'tsc',
+        'build:watch': 'tsc --watch',
+      },
+      // TODO(kschaaf): Version in lock-step with source?
+      version: pkgJson.version,
+      dependencies: {
+        // TODO(kschaaf): make component version range configurable?
+        [pkgJson.name!]: '^' + pkgJson.version!,
+        // TODO(kschaaf): make @lit-labs/react version configurable?
+        '@lit-labs/react': '^1.0.4',
+      },
+      peerDependencies: {
+        // TODO(kschaaf): make react version(s) configurable?
+        react: '^17.0.2',
+        '@types/react': '^17.0.19',
+      },
+      devDependencies: {
+        // TODO(kschaaf): make configurable?
+        typescript: '^4.3.5',
+      },
+      files: [...litModules.map(({module}) => module.jsPath)],
+    },
+    null,
+    2
+  );
+};
+
+const gitIgnoreTemplate = (litModules: LitModule[]) => {
+  return litModules.map(({module}) => module.jsPath).join('\n');
+};
+
+const tsconfigTemplate = () => {
+  return JSON.stringify(
+    {
+      compilerOptions: {
+        target: 'es2019',
+        module: 'es2015',
+        lib: ['es2020', 'DOM', 'DOM.Iterable'],
+        declaration: true,
+        declarationMap: true,
+        sourceMap: true,
+        inlineSources: true,
+        outDir: './',
+        rootDir: './src',
+        strict: true,
+        noUnusedLocals: true,
+        noUnusedParameters: true,
+        noImplicitReturns: true,
+        noFallthroughCasesInSwitch: true,
+        noImplicitAny: true,
+        noImplicitThis: true,
+        moduleResolution: 'node',
+        allowSyntheticDefaultImports: true,
+        experimentalDecorators: true,
+        noImplicitOverride: true,
+      },
+      include: ['src/**/*.ts'],
+      exclude: [],
+    },
+    null,
+    2
+  );
+};
+
+const wrapperModuleTemplate = (
+  packageJson: PackageJson,
+  moduleJsPath: string,
+  elements: LitElementDeclaration[]
+) => {
+  return javascript`
```

Comment:
We can make these template tags strip indentation.

Reply:
prettier will probably be less fussy